File: src/main/java/victor/training/cleancode/kata/videostore/autonom/Movie.java
package victor.training.cleancode.kata.videostore.autonom;

public class Movie {
  public enum Category {
    CHILDRENS,
    REGULAR,
    NEW_RELEASE
  }
  private String title;
  private Category priceCode;

  public Movie(String title, Category priceCode) {
    this.title = title;
    this.priceCode = priceCode;
  }

  public Category getPriceCode() {
    return priceCode;
  }

  public void setPriceCode(Category arg) {
    priceCode = arg;
  }

  public String getTitle() {
    return title;
  }
}

File: src/main/java/victor/training/cleancode/videostore/Movie.scala
package victor.training.cleancode.videostore

object Movie {
  val CHILDRENS = 2
  val REGULAR = 0
  val NEW_RELEASE = 1
}

class Movie(val _title: String, var _priceCode: Int) {

  def getPriceCode(): Int = {
    _priceCode
  }

  def setPriceCode(arg: Int): Unit = {
    _priceCode = arg
  }

  def getTitle(): String = {
    _title
  }
}